app/controllers/batches_controller.rb

```diff
@@ -12,7 +12,7 @@ def show
   end
 
   def detail
-    @batch_items = BatchItem.where(accession_number: params[:accession_number]).order(:updated_at)
+    @batch_items = BatchItem.where(accession_number: params[:accession_number]).order(:row_number)
```

Comment:
This doesn't work for me locally. Isn't the detail the accession number detail and the show is the view that lists the batch items by row?